package.json

```diff
@@ -9,7 +9,7 @@
     "babel-server": "babel server/ -d dist/server/ && npm run babel-app",
     "build": "npm run clean && mkdir dist/ && npm run babel-server && npm run babel-app",
     "clean": "rm -rf dist/",
-    "coveralls": "nyc --reporter=lcov --reporter=text --require babel-register  mocha server/tests/**/*.test.js --exit",
+    "coveralls": "nyc --reporter=lcov --reporter=text --require babel-register  mocha --timeout 15000 server/tests/**/*.test.js --exit",
```

Comment:
why do we need this here?

Reply:
After I resolve merge conflicts I discovered that coveralls wasn't reporting so I met Victor and he suggested I add the timeout. That got coveralls to show report.